config/runtime.go

```diff
@@ -23,13 +23,13 @@ import (
 // Runtime specifies the base runtime configuration fields that should be included in all witchcraft-server-go
 // server runtime configurations.
 type Runtime struct {
-	DebugConfig  DebugConfig        `yaml:"debug,omitempty"`
-	HealthChecks HealthChecksConfig `yaml:"health-checks,omitempty"`
-	LoggerConfig *LoggerConfig      `yaml:"logging,omitempty"`
+	DiagnosticsConfig DiagnosticsConfig  `yaml:"debug,omitempty"`
```

Comment:
Need to rename yaml field to `diagnostics`.

Reply:
oops, thanks!